modules/gridmap/editor/grid_map_editor_plugin.h

```diff
@@ -208,6 +242,7 @@ class GridMapEditor : public VBoxContainer {
 	void _set_clipboard_data();
 	void _update_paste_indicator();
 	void _do_paste();
+	void _show_viewports_transform_gizmo(bool value);
```

Comment:
```suggestion
	void _show_viewports_transform_gizmo(bool p_value);
```

Reply:
hello! thank you for reviewing the code. What is the fatest method to accept your changes?
 
I have a button "Commit suggestion" for each of your suggestions, but it would then split the PR into several commits, while you prefer it in a single commit.

Reply:
You can add suggestions to a batch and commit all together, then you'll need to squash your commits into one, see [the interactive rebase](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase), or you can apply them manually